src/identity.js

```diff
@@ -1056,42 +1056,36 @@ export default function Identity(mpInstance) {
                         mpInstance._Persistence.savePersistence(cookies, mpid);
                     }
 
-                    if (mpInstance._APIClient.shouldEnableBatching()) {
```

Comment:
Previously, UIC and UAC were only on batching.  Now that all are on batching, we don't need this check